### ISSUE TITLE
Create a "what you need" cargo.toml 

### ISSUE THREAD

**Issue by brimigs** [solana-playground/solana-playground#260]
Update the playground UI to have a file that creates your cargo.toml file of everything you would need to move your code over to a different IDE. I.e. after testing an anchor program in the solana playground, copying it over to vs code to use it in the create-solana-dapp scaffold and easily know all the dependencies you would need to do that.

**Comment by acheroncrypto** [comment]
> actually.. didn't realize there was an export button

Yes, using the Export button creates all files you need to use the program locally, including `Cargo.toml` with correct the dependencies and versions.

![export-button](https://github.com/user-attachments/assets/67f0e9b6-9d97-4523-b546-c2d803542735)

The reverse is also possible — you can drag & drop an entire Anchor project folder into Playground after clicking the Import button (to the left of the Export button).

**Comment by deauthe** [comment]
however I feel there should be a way for the developer to see the cargo.toml or something similar like a "dependencies"/"configuration" section where it's possible to see all of that without actually exporting the project.
